<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBMStarupEditPanel</class>
 <widget class="QWidget" name="DBMStarupEditPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBMStarupEditPanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="_dbTypeTag">
       <property name="text">
        <string>DB Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="_dbType">
       <item>
        <property name="text">
         <string>pgsql</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="_dbNameTag">
       <property name="text">
        <string>DB Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_dbName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="_dbIPTag">
       <property name="text">
        <string>DB IPAddr:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="_dbIP"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="_dbPortTag">
       <property name="text">
        <string>DB Port:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="_dbPort"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="_dbUserTag">
       <property name="text">
        <string>DB User</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="_dbUser"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="_dbPasswordTag">
       <property name="text">
        <string>DB Pwd:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="_dbPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="_outputTag">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
